tests/integration/basic_data/program/index.ts

```diff
@@ -0,0 +1 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
```

Comment:
Nit: Maybe add a comment that the Pulumi program doesn't _do_ anything, and that the tests just verify the properties of _all_ resources created. (Which in this case would just be the stack resource.)

Or perhaps just `console.log("[this Pulumi program intentionally left blank]");`?